src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -494,8 +494,16 @@ private async Task<bool> ShouldExitAfterDeterminingHostRuntime(bool isVerbose)
 
                 if (!string.Equals(HostRuntime, "default", StringComparison.OrdinalIgnoreCase))
                 {
-                    var isNet8InProcSpecified = string.Equals(HostRuntime, DotnetConstants.InProc8HostRuntime, StringComparison.OrdinalIgnoreCase);
-                    StartHostAsChildProcess(isNet8InProcSpecified);
+                    if (Utilities.IsMinifiedVersion())
+                    {
+                        ThrowIfInProc();
+                    }
+                    else
```

Comment:
nit: don't need else since an exception would've been thrown already

Reply:
good point, missed it before